src/modules/components/Dashboard/Cards/TotalStaked.tsx

```diff
@@ -23,88 +28,94 @@ const TotalStaked = () => {
   const { data: ggpStake } = useGetGGPStake(address)
   const { data: straightRatio } = useGetContractCollateralizationRatio(address)
   const { data: avaxMatched } = useGetAVAXAssigned(address)
-  const { data: ggpPrice } = useGetGGPPrice()
+  const { data: avaxStaked } = useGetAVAXStake(address)
 
   const { isOpen, onClose, onOpen } = useDisclosure()
   const { isOpen: isOpenUnstake, onClose: onCloseUnstake, onOpen: onOpenUnstake } = useDisclosure()
 
-  const ggpStakeInAVAX = ggpStake.mul(ggpPrice).div(BigNumber.from(10).pow(18))
-
   const stats = [
     {
-      name: 'GGP collateral ratio',
+      name: 'GGP COLLATERAL RATIO',
       stat: `${
-        straightRatio.eq(constants.MaxUint256) ? '∞' : Number(formatEther(straightRatio)).toFixed(2)
-      }%`,
+        straightRatio.eq(constants.MaxUint256)
+          ? '∞'
+          : Number(formatEther(straightRatio.mul(100))).toFixed(2)
+      } %`,
+      tooltip:
+        'Your GGP Collateralization ratio is the amount of GGP required to be able to match your 1000 AVAX. Expressed as a percentage of staked AVAX value.',
+    },
+    {
+      name: 'GGP STAKED',
+      stat: `${Number(formatEther(ggpStake)).toFixed(2)} GGP`,
+      tooltip: 'The total amount of GGP you have staked.',
+    },
+    {
+      name: 'AVAX STAKED',
+      stat: `${Number(formatEther(avaxStaked)).toFixed(2)} AVAX`,
+      tooltip:
+        'The total amount of AVAX you have deposited in the protocol. While your minipool is running you will be unable to withdraw this.',
     },
-    { name: 'Total GGP staked', stat: `${Number(formatEther(ggpStake)).toFixed(2)} GGP` },
     {
-      name: 'Total GGP staked converted to AVAX',
-      stat: `${Number(formatEther(ggpStakeInAVAX)).toFixed(2)} AVAX`,
+      name: 'MATCHED AMOUNT',
+      stat: `${Number(formatEther(avaxMatched)).toFixed(2)} AVAX`,
+      tooltip: 'The matched amount of AVAX received from the liquid staker pool.',
     },
-    { name: 'Matching', stat: `${Number(formatEther(avaxMatched)).toFixed(2)} AVAX` },
   ]
 
+  if (ggpStake.eq(0) || avaxMatched.eq(0)) {
+    return (
+      <DashboardEmptyCard icon={<PiVault color={'#3E33BB'} size={40} />} title="My Stake">
```

Comment:
```suggestion
      <DashboardEmptyCard icon={<PiVault color={colors.blue} size={40} />} title="My Stake">
```
well idk how to do colors but raw `#` strings suck

Reply:
Will fix this in later issue, I need to make changes across the app here.